src/components/lobby/RoomsList.js

```diff
@@ -68,10 +88,15 @@ const RoomsListItem = ({ room: { gameID, players }, game, onJoin, user }) => {
           <Label as="span" style={{ marginLeft: "1rem" }}>
             #<Label.Detail>{gameID}</Label.Detail>
           </Label>
+          {current && (
+            <Label as="span" style={{ marginLeft: "1rem" }} color="green">
```

Comment:
Can we make all 3 green things (username, "your game" and join button) be used to join the room? Because every green thing in our application is clickable so it would be consistent